### PR TITLE
Allow to configure access/error log files

### PR DESCRIPTION
**Pull request by bbinet**

### CODE REVIEW

File: nginx/files/_log.conf [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{%- from "nginx/map.jinja" import server with context -%}
+{%- set site = salt['pillar.get']('nginx:server:site:'+site_name) %}
 
-  access_log /var/log/nginx/{{ site_name }}-access.log;
-  error_log /var/log/nginx/{{ site_name }}-error.log;
+  access_log {{ site.get('access_log', server.log_dir + '/' + site_name + '-access.log') }};
```
Comment (fpytloun) [inline nginx/files/_log.conf]: I would prefer this structure of metadata:

```
site:
  log:
    access: 
    error: 
```

Comment (bbinet) [reply nginx/files/_log.conf]: Ok, I can change that.

And for the main server logs, should I use the same metadata structure?

Comment (fpytloun) [reply nginx/files/_log.conf]: Yes, that would be great

Comment (bbinet) [reply nginx/files/_log.conf]: The structure of log metadata is now updated with commit: d63beda

**Comment by fpytloun** [comment]
Thank you

**Pull request merged**